files/manage_bootstrap_user.py

```diff
@@ -7,24 +7,24 @@
 import re
 import sys
 
+new_path = [os.path.join(os.getcwd(), "lib")]
```

Comment:
This should hopefully change once galaxy is installable, but I guess we'll need this for now.